### ISSUE TITLE
Why does the new cps.csv.gz file contain many large negative values for the other_ben variable?

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/taxdata#186]
```
$ gunzip -k cps.csv.gz

$ ls -l cps.csv*
-rw-r--r--  1 mrh  staff  82216905 Apr 27 08:39 cps.csv
-rw-r--r--  1 mrh  staff  15088523 Apr 27 08:39 cps.csv.gz

$ ../csv_vars.sh cps.csv | grep _ben
24 ssi_ben
25 vet_ben
26 snap_ben
27 mcare_ben
28 mcaid_ben
30 tanf_ben
31 housing_ben
32 wic_ben
65 other_ben

$ awk -F, 'NR>1{t++}$65<0{n++;if($65<m)m=$65}END{print t,n,100*n/t,m}' cps.csv
456465 102821 22.5255 -36165
```
More than one in five have negative `other_ben` values, with the one with the largest absolute value being minus thirty-six thousand dollars.

None of the other benefits had any negative values (in tabulations not shown here).

Do I have the wrong file?  @andersonfrailey, can you replicate these results?

**Comment by andersonfrailey** [comment]
@martinholmer I was able to replicate your results. I'm speculating, but I think this was caused by my multiplying the weights in the CPS file by 100 at the beginning of the final prep scripts. This would cause the values of the `tanf` and `wic` variables we subtract from the `other_ben` initial sum in the `benefits` function of `finalprep.py` to be exceptionally large.

I should be able to fix it by simply moving the line of code where I multiply the CPS weights by 100 to the end of the script. Will get a PR up soon.
